src/Action.hs

```diff
@@ -507,6 +507,13 @@ topicInRefinementTimedOut = topicTimeout RefinementPhaseTimeOut
 topicInVotingTimedOut :: (ActionPhaseChange m) => AUID Topic -> m ()
 topicInVotingTimedOut = topicTimeout VotingPhaseTimeOut
 
+topicInVotingSetbackTopicToJuryPhase
```

Comment:
Would it make sense to have a function `dangerousChangePhase`?  Then we could the two force-timeouts, the rest from voting to jury phase, and everything else that we may find we have a need of.